content/enterprise_influxdb/v1.9/troubleshooting/frequently-asked-questions.md

```diff
@@ -289,7 +304,8 @@ Acceptable Boolean syntax differs for data writes and data queries.
 
 For example, `SELECT * FROM "hamlet" WHERE "bool"=True` returns all points with `bool` set to `TRUE`, but `SELECT * FROM "hamlet" WHERE "bool"=T` returns nothing.
 
-{{% warn %}} [GitHub Issue #3939](https://github.com/influxdb/influxdb/issues/3939) {{% /warn %}}
+<!-- TODO: closed issue. Edit docs if necessary. -->
```

Comment:
nice! Are we tracking issues for potential edits or have a todo list for?

Reply:
https://github.com/influxdata/docs-v2/issues/2630